app/src/main/java/com/mapbox/dash/example/MainApplication.kt

```diff
@@ -29,6 +29,14 @@ class MainApplication : Application() {
                 // Example of a debug option that teleports the user to a specific location at each app launch.
                 defaultLocation = MAPBOX_DC_OFFICE
             }
+            routeOptionsConfig {
+                avoidHighways = true
+                avoidFerries = true
+                avoidTolls = true
+            }
+            uiSettingsConfig {
+                showRouteOptions = false
```

Comment:
let's show route options by default and keep the option to hide that in the side menu

Reply:
@RingerJK done.